setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
+
+setup(
+    name='floorist',
+    version='0.0.1',
+    description='Helper for dumping SQL queries from a PostgreSQL database into S3 buckets in parquet format',
+    url='https://github.com/RedHatInsights/floorist',
+    author='Halász Dávid (@skateman)',
+    author_email='',
```

Comment:
@skateman  there's no public email anywhere in your GH profile so I didn't want to mess up with your privacy ;) - this is also optional BTW , just say it and I'll remove the attribute

Reply:
wooops, done